doc/index.html

```diff
@@ -1,163 +1,163 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<!DOCTYPE html
-     PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
-     "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
-<head>
-<title>libexecstream home page</title>
-<link href="libexecstream.css" type="text/css" rel="stylesheet"></link>
-</head>
-
-<body>
-
-<h1>The libexecstream library</h1>
-
-<table class="maintable">
-<col class="linksbar"></col>
-<col class="body"></col>
-<tr>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We could move this to a `gh-pages` branch and serve the docs directly here on github